### PR TITLE
rococo: update weights

### PR DESCRIPTION
**Pull request by coderobe**
this is a full run of the rococo weights benchmarks using the same process we use for the other runtimes.
this covers the benchmarks mentioned in https://github.com/paritytech/polkadot/issues/5181

### CODE REVIEW

**Comment by coderobe** [comment]
the full benchmark log is available at https://gitlab.parity.io/parity/polkadot/-/jobs/1475199/raw

**Comment by coderobe** [comment]
Benchmark failure(s):

```
2022-04-03 05:12:32 Running Benchmark: runtime_parachains::paras_inherent.enter_bitfields 1/50 1/1    
2022-04-03 05:12:32 Took active validators from set with wrong size    
2022-04-03 05:12:32 Took active validators from set with wrong size.    
2022-04-03 05:12:32 Took active validators from set with wrong size    
2022-04-03 05:12:32 Took active validators from set with wrong size    
2022-04-03 05:12:32 Took active validators from set with wrong size.    
2022-04-03 05:12:32 Took active validators from set with wrong size    
2022-04-03 05:12:32 Took active validators from set with wrong size    
2022-04-03 05:12:32 Took active validators from set with wrong size.    
2022-04-03 05:12:32 Took active validators from set with wrong size    
2022-04-03 05:12:32 panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `101`', /home/gitlab-runner/builds/G3WgZWf2/0/parity/polkadot/runtime/parachains/src/paras_inherent/benchmarking.rs:119:9    
Error: 
   0: [91mInvalid input: Error executing and verifying runtime benchmark: Execution aborted due to trap: wasm trap: wasm `unreachable` instruction executed
      WASM backtrace:

          0: 0x67a5 - <unknown>!rust_begin_unwind
          1: 0x2f6a - <unknown>!core::panicking::panic_fmt::h8f11323637b4db3b
          2: 0x703e - <unknown>!core::panicking::assert_failed_inner::hb7fa033f7ee539c7
          3: 0x4410e - <unknown>!core::panicking::assert_failed::hf837eae46cefa84c
          4: 0x2fc664 - <unknown>!<rococo_runtime::Runtime as frame_benchmarking::utils::runtime_decl_for_Benchmark::Benchmark<sp_runtime::generic::block::Block<sp_runtime::generic::header::Header<u32,sp_runtime::traits::BlakeTwo256>,sp_runtime::generic::unchecked_extrinsic::UncheckedExtrinsic<sp_runtime::multiaddress::MultiAddress<<<sp_runtime::MultiSignature as sp_runtime::traits::Verify>::Signer as sp_runtime::traits::IdentifyAccount>::AccountId,()>,rococo_runtime::Call,sp_runtime::MultiSignature,(frame_system::extensions::check_non_zero_sender::CheckNonZeroSender<rococo_runtime::Runtime>,frame_system::extensions::check_spec_version::CheckSpecVersion<rococo_runtime::Runtime>,frame_system::extensions::check_tx_version::CheckTxVersion<rococo_runtime::Runtime>,frame_system::extensions::check_genesis::CheckGenesis<rococo_runtime::Runtime>,frame_system::extensions::check_mortality::CheckMortality<rococo_runtime::Runtime>,frame_system::extensions::check_nonce::CheckNonce<rococo_runtime::Runtime>,frame_system::extensions::check_weight::CheckWeight<rococo_runtime::Runtime>,pallet_transaction_payment::ChargeTransactionPayment<rococo_runtime::Runtime>)>>>>::dispatch_benchmark::hbc0648726bb7873a
          5: 0x395cfe - <unknown>!Benchmark_dispatch_benchmark
   0: [0m

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
```

/cc @brunopgalvao @shawntabrizi

**Comment by brunopgalvao** [comment]
Looks like we have to uncomment [this](https://github.com/paritytech/polkadot/blob/6693934899d62ff6e07b323db15c4a81dbb507d9/runtime/parachains/src/paras_inherent/benchmarking.rs#L102) line for Rococo.

**Comment by coderobe** [comment]
that's not too great

**Comment by ggwpez** [comment]
Okay I will take a look, thanks @coderobe.  
As for the failing one: This should be the last failing benchmark across all benchmarks and runtimes.  
So we can probably already merge the weights if they are sane and then fix the last one.

File: runtime/rococo/src/weights/frame_system.rs [hunk 1]
```diff
@@ -43,8 +43,10 @@ use sp_std::marker::PhantomData;
 /// Weight functions for `frame_system`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> frame_system::WeightInfo for WeightInfo<T> {
-	fn remark(_b: u32, ) -> Weight {
+	fn remark(b: u32, ) -> Weight {
```
Comment (ggwpez) [inline runtime/rococo/src/weights/frame_system.rs:46]: Weird that this is not ignore here.   
For all other runtimes it also is the case.

File: runtime/rococo/constants/src/weights/block_weights.rs [hunk 2]
```diff
@@ -47,16 +43,16 @@ parameter_types! {
...
 	///
 	/// Percentiles [NS]:
-	///   99th: 3_266_514
-	///   95th: 3_205_264
-	///   75th: 3_101_379
-	pub const BlockExecutionWeight: Weight = 3_098_278 * WEIGHT_PER_NANOS;
+	///   99th: 3_855_334
+	///   95th: 3_815_093
+	///   75th: 3_762_125
+	pub const BlockExecutionWeight: Weight = 3_745_959 * WEIGHT_PER_NANOS;
```
Comment (ggwpez) [inline runtime/rococo/constants/src/weights/block_weights.rs:55]: This is a 20% increase, thats not too good.  
I assume the CI benchmarking machine is a bit slower somehow.  

Do you think we can still merge this @shawntabrizi ?

**Review by ggwpez — APPROVED** [review]

File: runtime/rococo/src/weights/frame_system.rs (hunk 1 above)
Comment (shawntabrizi) [reply runtime/rococo/src/weights/frame_system.rs:46]: `1_000` is literally the smallest value that can be returned from the benchmarking tool. (representing 1 NS)

This is just noise, and potentially we could round these things down to zero, but I see no problem with there being something here, and it being slightly inconsistent. It is negligible.

File: runtime/rococo/constants/src/weights/block_weights.rs (hunk 2 above)
Comment (shawntabrizi) [reply runtime/rococo/constants/src/weights/block_weights.rs:55]: If you think it is slower, we should re-run it.

But if these are the result, then these are the results...

End of the day, this is an increase of 700 micro-seconds:

(700 microseconds) / (2 seconds) =
0.00035 = .035%

It doesn't matter?

**Review by shawntabrizi — APPROVED** [review]

**Pull request merged**